src/extension.ts

```
@@ -425,6 +616,214 @@ console.log('Roo: After registering runserver command');
 							keys: allKeys,
 						});
 						return;
+					case 'getProxies(request)':
+						// Get proxies from ProxyPoolManager instead of legacy storage
+						const availableProxies = proxyPoolManager?.getAllProxies() || [];
+						panel.webview.postMessage({
+							command: "getProxies(response)",
+							proxies: availableProxies,
+						});
+						return;
+					case 'getProxyAssignments(request)':
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)